### DISCUSSION TITLE
How to speed up catching up blocks?

### DISCUSSION THREAD

**Discussion by formingform** [blockscout/blockscout#9528] [Q&A]
I've had my test chain running for a while, now I luanched blockscout, so it began to catch up blocks. I want to ask is there any best practice to speed up catching up blocks?
Thanks

**Comment by formingform** [comment]
Are there any parameters that affect this process?

**Reply by GimluCom** [reply]
https://docs.blockscout.com/about/faqs#how-do-i-speed-up-my-self-hosted-instance
